scandipwa/packages/demo-widget/src/component/DemoWidget/DemoWidget.component.js

```diff
@@ -59,12 +66,50 @@ export class DemoWidgetComponent extends PureComponent {
         return <Html content={ content } />;
     }
 
-    // setBackgroundColor() {
-    //     const { color, widgetRef } = this.props;
-    // }
+    renderTimeBlock(time, suffix) {
+        return (
+            <div
+              block="Countdown"
+              elem="TimeBlock"
+            >
+                <span>{ `${time} ${suffix}` }</span>
+            </div>
+        );
+    }
+
+    renderLink() {
+        const { link } = this.props;
+        return (
+            <div
+              block="DemoWidget"
+              elem="Text"
+            >
+                { !!link
```

Comment:
The `!!` seems too complex, and what is the value we get for the link? Is it string or null?
Maybe we can return empty string if it is not set and then check for it in the FE code